composer.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Jcowie/FeatureToggle",
+  "description": "This module provides support for Feature Toggles within Magento 2",
+  "license": "MIT",
+  "require": {
+    
+  },
+  "autoload": {
+    "files": [
+      "registration.php"
+    ],
+    "psr-4": {
+      "Jcowie\\FeatureToggle\\": ""
+    }
+  },
+  "minimum-stability": "beta",
+  "type": "magento2-module",
+  "extra": {
```

Comment:
I dont think M2 requires the mapping anymore. But again I dont think that it hurts atm to have in present.